package.json

```diff
@@ -1,10 +1,11 @@
 {
+  "private": true,
+  "workspaces": [
+    "api",
+    "app"
+  ],
   "devDependencies": {
-    "@anolilab/multi-semantic-release": "^1.1.10",
-    "gradle-semantic-release-plugin": "^1.10.1",
-    "@saithodev/semantic-release-backmerge": "^4.0.1",
-    "@semantic-release/changelog": "^6.0.3",
-    "@semantic-release/exec": "^6.0.3",
-    "@semantic-release/git": "^10.0.1"
+    "@anolilab/multi-semantic-release": "^1.1.12",
+    "semantic-release": "^22.0.12"
```

Comment:
Can we bump semrel to the latest version of 24.x?

Although I think some of the plugins don't support it so the safest upgrade would be 22.0 -> 23.0 since that's just bumping the required node to 20

Reply:
@brosssh Actually, I just checked the change log, doesn't seem to include any major breaking changes that's relevant to us. You should be able to upgrade to v24/v25 beta just fine assuming dependency support it.